types/node/index.d.ts

```diff
@@ -1518,43 +1484,65 @@ declare module "https" {
     }
 
     export interface RequestOptions extends http.RequestOptions {
-        pfx?: any;
-        key?: any;
+        pfx?: string | Buffer;
+        key?: string | Buffer;
         passphrase?: string;
-        cert?: any;
-        ca?: any;
+        cert?: string | Buffer;
+        ca?: string | Buffer | string[] | Buffer[];
```

Comment:
`ca` matches but `cert` does not. Seems I was interrupted too often during my review...